app/js/tests/reducers/test_form_content.js

```diff
@@ -42,5 +39,37 @@ test( 'CHANGE_CONTENT changes the field', function ( t ) {
 	deepFreeze( stateBefore );
 	t.deepEqual( formContent( stateBefore, action ), expectedState );
 	t.end();
+} );
+
+test( 'CHANGE_CONTENT throws an error if the field name is not allowed', function ( t ) {
+	var action = { type: 'CHANGE_CONTENT', payload: { value: 'supercalifragilistic', contentName: 'unknownField' } };
 
+	t.throws( function () {
+		formContent( {}, action );
+	} );
+	t.end();
 } );
+
+test( 'INITIALIZE_CONTENT changes multiple fields', function ( t ) {
+	var stateBefore = { paymentType: 'PPL', amount: 0, recurringPayment: 0 },
+		expectedState = { paymentType: 'BEZ', amount: '25,00', recurringPayment: 0 },
+		action = { type: 'INITIALIZE_CONTENT', payload: { amount: '25,00', paymentType: 'BEZ' } };
+
+	deepFreeze( stateBefore );
```

Comment:
To assert that the state is immutable. If the state is somehow changed in the reducer (instead of creating a new state object), the deep frozen object will throw an exception.